internal/js-dependency-scanner/src/main/java/org/wso2/security/tools/scanner/dependency/js/utils/ConfigParser.java

```diff
@@ -0,0 +1,248 @@
+    }
+
+    /**
+     * Parse jira credential details (Username, Password).
+     *
+     * @throws ConfigParserException exception occurred while parsing configuration details.
+     */
+    public static IssueCreator parseIssueTicketCreatorCredentials() throws ConfigParserException {
+
+        Properties properties = new Properties();
+        ClassLoader classLoader = ConfigParser.class.getClassLoader();
+        try (InputStream input = classLoader.getResourceAsStream(JSScannerConstants.ISSUECREATOR_CONFIG_FILE)) {
+            properties.load(input);
+        } catch (IOException e) {
+            throw new ConfigParserException("Error occurred in parsing JIRA Credentials : " + e);
+        }
+
+        IssueCreator issueCreatorAPI = new JIRAIssueCreator(new JIRARestClient(), properties.getProperty
+                (JSScannerConstants.USERNAME).toCharArray(),
+                properties.getProperty(JSScannerConstants.PASSWORD).toCharArray(),
+                properties.getProperty(IssueCreatorConstants.WSO2_JIRA_BASE_URL));
+        HashMap<String, String> ticketAssigneeMapper = new HashMap<>();
+        //assignees for each products
+        ticketAssigneeMapper.put(JSScannerConstants.AM, properties.getProperty(IssueCreatorConstants.APIM));
+        ticketAssigneeMapper.put(JSScannerConstants.IDENTITYSERVER, properties.getProperty(
+                IssueCreatorConstants.IDENTITYSERVER));
+        ticketAssigneeMapper.put(JSScannerConstants.INTEGRATION, properties.getProperty(IssueCreatorConstants
+                .INTEGRATION));
+        ticketAssigneeMapper.put(JSScannerConstants.STREAMPROCESSOR, properties.getProperty(IssueCreatorConstants
+                .STREAMPROCESSOR));
+        ticketAssigneeMapper.put(JSScannerConstants.OB, properties.getProperty(IssueCreatorConstants.OPENBANKING));
+        issueCreatorAPI.setAssigneeMapper(ticketAssigneeMapper);
+        return issueCreatorAPI;
+
+    }
+
+    /**
+     * parse JIRA ticket information.
+     *
+     * @throws ConfigParserException exception occurred while parsing configuration details.
+     */
+    public static void parseJIRATicketInfo() throws ConfigParserException {
+        Properties properties = new Properties();
+        ClassLoader classLoader = ConfigParser.class.getClassLoader();
+        try (InputStream input = classLoader.getResourceAsStream(JSScannerConstants.JIRA_TICKET_INFO_FILE)) {
+            properties.load(input);
+        } catch (IOException e) {
+            throw new ConfigParserException("Error occurred in parsing JIRA Credentials : " + e);
+        }
+        JIRAIssueCreator.setTicketSubject(properties.getProperty(IssueCreatorConstants.TICKET_SUBJECT));
+        JIRAIssueCreator.setProjectKey(properties.getProperty(IssueCreatorConstants.PROJECT_KEY));
+        JIRAIssueCreator.setIssueLabel(properties.getProperty(IssueCreatorConstants.ISSUELABEL));
+        JIRAIssueCreator.setIssueType(properties.getProperty(IssueCreatorConstants.ISSUE_TYPE));
```

Comment:
All of the above depends on property loading being successful. Therefore they should go inside the try block